### PR TITLE
Add more composer scripts

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,6 +15,27 @@
 		"guzzlehttp/promises": "~1.0",
 		"psr/log": "~1.0"
 	},
+	"require-dev": {
+		"phpunit/phpunit": "~4.8.0|~5.3.0",
+		"jakub-onderka/php-parallel-lint": "0.9.2"
+	},
+	"suggest": {
+		"etsy/phan": "Allows running static analysis on the package (requires PHP 7+)"
```
Comment (JeroenDeDauw) [inline composer.json:23]: Suggest is meant for packages using the one where you put it if I'm not mistaken. This one does not seem to make a lot of sense like that

Comment (addshore) [reply composer.json:23]: So, I was trying to figure out how to do this.
I was phan running on CI, but I can't add it to the composer.json as it specifies php 7.0+, so this package would then not be installable with php 5.5 etc..
I put it in suggest as this repo does indeed have a phan config and you could use it..
for CI I guess I will add a step that requires phan in cases where PHP is greater than 7?
Or maybe make a separate job just for phan and have a larger build matrix..

Comment (JeroenDeDauw) [reply composer.json:23]: Both CI options seem sane to me. Idk a good solution for the non-CI case though. Never seen this situation before

(silently shouts "just switch to PHP 7 already")

Comment (addshore) [reply composer.json:23]: "just switch to PHP 7 already"
I know, but not yet! ;)

Comment (samwilson) [reply composer.json:23]: I think it's okay to put phan as a suggestion. `suggest` [is for](https://getcomposer.org/doc/04-schema.md#suggest):

> Suggested packages that can enhance or work well with this package. These are just informational and are displayed after the package is installed, to give your users a hint that they could add more packages, even though they are not strictly required.

It's a shame there isn't a `suggest-dev`.

Comment (addshore) [reply composer.json:23]: > It's a shame there isn't a suggest-dev.

Yeh, that would be ideal ;)
I'll knock something up to make phan run in CI for php7 etc at some point too!

**Pull request merged**